src/main/java/com/itschool/eventmanagment/services/EventServiceImp.java

```diff
@@ -59,9 +59,9 @@ public List<EventDTO> getEvents() {
     public List<EventDTO> getEvents(LocalDateTime to, LocalDateTime from, String location, String sortBy) {
         List<Event> events;
         if ("location".equalsIgnoreCase(sortBy)) {
-            events = eventRepository.findByDateTimeBetweenAndLocationContainingIgnoreCaseOrderByLocationAsc(to, from, location);
+            events = eventRepository.findByEventDateTimeBetweenAndLocationContainingIgnoreCaseOrderByLocationAsc(to, from, location);
         } else {
-            events = eventRepository.findByDateTimeBetweenAndLocationContainingIgnoreCaseOrderByDateTime(to, from, location);
+            events = eventRepository.findByEventDateTimeBetweenAndLocationContainingIgnoreCaseOrderByEventDateTime(to, from, location);
         }
         return events.stream().map(event -> objectMapper.convertValue(event, EventDTO.class)).collect(Collectors.toList());
```

Comment:
Can we maybe simplify a bit this stream using just `toList()` method?

```suggestion
        return events.stream().map(event -> objectMapper.convertValue(event, EventDTO.class)).toList();
```